apps/src/templates/census2017/CensusMap.jsx

```diff
@@ -0,0 +1,345 @@
+/* global google adjustScroll*/
+import $ from 'jquery';
+import React, { PropTypes, Component } from 'react';
+import ReactDOM from 'react-dom';
+
+class CensusMapInfoWindow extends Component {
+  static propTypes = {
+    handleTakeSurveyClick: PropTypes.func.isRequired,
+    schoolId: PropTypes.string.isRequired,
+    schoolName: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    state: PropTypes.string.isRequired,
+    location: PropTypes.string.isRequired,
+    teachesCs: PropTypes.string.isRequired
+  };
+
+  render() {
+    let censusMessage;
+
+    switch (this.props.teachesCs) {
+      case 'YES':
+        censusMessage = "We believe this school offers Computer Science.";
```

Comment:
Future cleanup: These census messages should be localized.

Reply:
Agreed. There are a ton of stings on the /yourschool page that aren't localized.